src/SpotifyApi.NetCore.Tests/FollowApiTests.cs

```diff
@@ -102,30 +79,23 @@ public async Task FollowArtists_Unfollowed_CheckUserFollowsTrue()
             // arrange
             string[] artistIds = new string[] { "74ASZWbe4lXaubB36ztrGX" };
 
-            var http = new HttpClient();
-            IConfiguration config = TestsHelper.GetLocalConfig();
-            string accessToken = config["SpotifyUserBearerAccessToken"];
-            var accounts = new AccountsService(http, config);
-
-            var api = new FollowApi(http, accounts);
-
             // unfollow
-            await api.UnfollowArtists(artistIds, accessToken: accessToken);
+            await api.UnfollowArtists(artistIds, bearerAccessToken);
 
-            // checking if artists were unfollowed successfully
+            // checking if artists are already followed
             Assert.IsFalse((await api.CheckCurrentUserFollowsArtists(
                 artistIds,
-                accessToken: accessToken)).First(), 
-                "Artist should have been unfollowed at this point");
+                bearerAccessToken)).First(), 
+                "Artist is already followed.");
 
             // act
-            await api.FollowArtists(artistIds, accessToken: accessToken);
+            await api.FollowArtists(artistIds, bearerAccessToken);
 
             // assert
-            // checking if artists were followed successfully
+            // checking if artists were unfollowed successfully
```

Comment:
Comment is wrong, why did you change it? Here we are checking to see if the artist was followed successfully.